datarepo-clienttests/README.md

```diff
@@ -409,12 +412,30 @@ cat build/reports/spotbugs/main.txt
 ```
 
 ## Troubleshooting
-* Check that the server specification file property of the test configuration points to the correct URL you want to test
-against.
 
-* Check that your IP address is included on the IP whitelist for the cluster you're testing against. The easiest way to
+#### IP Whitelist
+Check that your IP address is included on the IP whitelist for the cluster you're testing against. The easiest way to
 do this is to connect to the Non-Split Broad VPN, because the VPN IP addresses are already included on the IP whitelist
 for the Jade dev cluster.
 
-* Check that you are calling the correct version of Gradle (6.1.1 or higher). Use the Gradle wrapper in the sub-project
+#### Gradle version
+Check that you are calling the correct version of Gradle (6.1.1 or higher). Use the Gradle wrapper in the sub-project
 (`.gradlew`), not the one in the parent directory (`../.gradlew`).
```

Comment:
FWIW, I think we're on the higher version of gradle in the main project.  Meta note: it's super confusing to have a standalone gradle project within another gradle project.  This should probably be moved to its own repo or become a submodule of the main project in here

Reply:
Good to know about the Gradle version, thanks!

Agreed about the standalone Gradle project. It doesn't always import nicely in IntelliJ without futzing either, which I find really frustrating. I'm currently working part-time with QA to move Test Runner to its own [library](https://github.com/DataBiosphere/terra-test-runner). You can see progress so far in this Data Repo [branch](https://github.com/DataBiosphere/jade-data-repo/pull/718). Part of this effort is moving it to a Gradle sub-project.

Reply:
Oh that's great!